### ISSUE TITLE
[Crash][1.20.1][NeoForge] Crash when updating from 1.5.2 to 1.5.3

### ISSUE THREAD

**Issue by drakray** [Jusey1/Jewelcraft#2]
https://pastebin.com/mFUeZayS

ModList at end of file

<details>
  <summary>Crash excerpt</summary>
  
```
Description: Exception ticking world

java.lang.NullPointerException: Cannot invoke "net.minecraft.world.entity.Entity.m_20185_()" because "p_45931_" is null
	at net.minecraft.world.level.EntityGetter.m_45930_(EntityGetter.java:97) ~[client-1.20.1-20230612.114412-srg.jar%23758!/:?] {re:mixin,re:classloading}
	at net.salju.jewelcraft.events.JewelcraftEvents.onCrops(JewelcraftEvents.java:259) ~[jewelry-1.5.3.jar%23623!/:?] {re:classloading}
	at net.salju.jewelcraft.events.__JewelcraftEvents_onCrops_Pre.invoke(.dynamic) ~[jewelry-1.5.3.jar%23623!/:?] {re:classloading,pl:eventbus:B}
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:73) ~[eventbus-6.0.5.jar%2389!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:315) ~[eventbus-6.0.5.jar%2389!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:296) ~[eventbus-6.0.5.jar%2389!/:?] {}
```
  
</details>

**Comment by drakray** [comment]
More Info: Sodalite Amulet was in my Charm Slot when I tried to open the world

**Comment by Jusey1** [comment]
Fixed in 1.5.4, sorry about that.

**Comment by drakray** [comment]
No problem, thank you for your good work :D
